interfaces/Portalicious/src/app/domains/project/project.api.service.ts

```diff
@@ -136,9 +136,8 @@ export class ProjectApiService extends DomainApiService {
   }): Promise<unknown> {
     return this.httpWrapperService.perform121ServiceRequest<Project>({
       method: 'POST',
-      endpoint: `${BASE_ENDPOINT}/${projectId().toString()}/notes`,
+      endpoint: `${BASE_ENDPOINT}/${projectId().toString()}/registrations/${registrationReferenceId().toString()}/note`,
```

Comment:
```suggestion
      endpoint: `${BASE_ENDPOINT}/${projectId().toString()}/registrations/${registrationReferenceId()}/note`,
```

The `.toString()`-part is only necessary for the `projectId`-specifically, as that is of the type `number`, it needs to be used as a `string`. The `registrationRefecendeId` would already be a string.

But in practice, its the same. ;)